reports/sdks.go

```diff
@@ -28,6 +29,25 @@ var (
 			FeatureRegex: regexp.MustCompile(`web5\.sdk\.\w+.Web5TestVectors(\w+)`),
 			VectorRegex:  regexp.MustCompile(`(\w+)\(\)`),
 			VectorPath:   "test-vectors",
+			Type:         "web5",
+		},
+		{
+			Name:         "tbdex-js",
+			Repo:         "TBD54566975/tbdex-js",
+			ArtifactName: "junit-results",
+			FeatureRegex: regexp.MustCompile(`TbdexTestVectors(\w+)`),
```

Comment:
so you have been naming your tests:
TbdexTestVectorsClassFeature ?

for example: 
`TbdexTestVectorsRfqCreate` ?

Reply:
Ah yep, Think I have a copy paste error for the vector regex though!